app/main.py

```diff
@@ -1 +1,38 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None :
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        beg = "{"
+        end = "}"
+        return f"{beg}Name: {self.name}, " \
+            f"Health: {self.health}, " \
+            f"Hidden: {self.hidden}{end}"
```

Comment:
The `__repr__` method can be improved by using a single f-string for the entire return statement. This enhances readability and performance. Consider using: `return f"{{Name: {self.name}, Health: {self.health}, Hidden: {self.hidden}}}"`.